test/testutil.py

```diff
@@ -147,3 +149,270 @@ def create_filesystem_customizations(rootfs: str):
     "-v", "/var/lib/containers/storage:/var/lib/containers/storage",
     "--security-opt", "label=type:unconfined_t",
 ]
+
+
+def get_ip_from_default_route():
+    default_route = subprocess.run([
+        "ip",
+        "route",
+        "list",
+        "default"
+    ], check=True, capture_output=True).stdout
```

Comment:
(nitpick) when using `text=True` here the `decode("utf-8")` should not be necessary (as the output is captured as text already)